### DISCUSSION TITLE
How to update dialog field based on the selection from another dialog field Lookup

### DISCUSSION THREAD

**Discussion by waleed51** [serenity-is/Serenity#5664] [Help] [answered]
Hi, 
I have a lookup and I want to update another dialog field based on the selected value from this lookup ?? can anyone help me ? 
and thank you.

**Comment by edwardch** [comment, marked as answer]
You should try use changeSelect2 like from [the Northwind demo code](https://github.com/serenity-is/Serene/blob/fe41ee63905f51b7ac1333ce0c3cce118ae5372f/Serene/Serene.Core/Modules/Northwind/OrderDetail/OrderDetailDialog.ts#L17).

**Reply by waleed51** [reply]
thank you
